allm_example.py

```diff
@@ -41,9 +41,9 @@
 import shutil
 
 # where the text files are
-anythingllm_summarydir = '/Users/aaronmccloud/Downloads/latestanythingllmcode'
+anythingllm_summarydir = '/home/frog/repos/latest-anything-llm'
```

Comment:
Let's move this to a `config.js`, or something like that, so we don't keep changing it back and forth.